### DISCUSSION TITLE
No recent NPM releases?

### DISCUSSION THREAD

**Discussion by rabrowne85** [sfatihk/electron-tray-window#17] [General]
@sfatihk have you published any new versions of this package? NPM is reporting 1.2.5 as the latest, but i think there have been a few pull requests merged since then and just wondering if a new version can be published?

**Comment by sfatihk** [comment]
Hi @rabrowne85. Years later, a new version(**1.2.7**) was published 🎉 😅.

https://www.npmjs.com/package/electron-tray-window

**Reply by rabrowne85** [reply]
Thanks @sfatihk for pushing the new release 👍
